File: Policlinica.Pacient.Angular/Models/LoginPacient.cs
using System;
using System.Collections.Generic;

namespace Policlinica.Pacient.Angular.Models
{
    public partial class LoginPacient
    {
        public int FkPacientId { get; set; }
        public string UserName { get; set; }
        public string Parola { get; set; }
        public DateTime DataInregistrare { get; set; }

        public Pacient FkPacient { get; set; }
    }
}

File: Policlinica.Pacient.Angular/ClientApp/src/app/models/login.pacient.ts
export class LoginPacient{
    FkPacientId:Number;
    UserName:string;
    Parola:String;
    DataInregistrare:Date;
    constructor(username:string,password:string){
           
            this.UserName=username;
            this.Parola=password;
    }
}